### DISCUSSION TITLE
Discover devices with definition

### DISCUSSION THREAD

**Discussion by atrovato** [Koenkk/zigbee2mqtt#8980] [Devices]
Hi,
is there a way to ask for all paired devices with definition and exposes over MQTT ?
(then get message over `zigbee2mqtt/brigde/devices` topic)

Thank you

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/information/mqtt_topics_and_message_structure.html#zigbee2mqttbridgedevices

**Reply by atrovato** [reply]
I saw it, but I don't want to plug/unplug my device to get this message.
I think it can be useful to explicitly ask for it.
Like a `zigbee2mqtt/brigde/devices/get` topic.

**Reply by Koenkk** [reply]
This is a retained MQTT message so your application will always have access to it

**Reply by atrovato** [reply]
Hoho that's great!
I just checked it, you're wonderful!
Thank you :D
